src/pages/myGeotab/whatsNew.tsx

```diff
@@ -1,8 +1,3045 @@
+			</li>
+		</ul>
+		<h2>GetFeed DebugData</h2>
+		<ul>
+			<li>
+				<a href="/sdk/software/api/reference/#GetFeed1">GetFeed</a>: <a href="/sdk/software/api/reference/#DebugData">DebugData</a> - Fixed sort order issue leading to possible missed records.
+			</li>
+		</ul>
+		<h2>FuelTransaction</h2>
+		<ul>
+			<li>
+				<a href="/sdk/software/api/reference/#FuelTransaction">FuelTransaction</a> - Added <code className="small-code-sample">Device</code> and <code className="small-code-sample">Driver</code>. These properties add fuel transactions for a device or user, rather than a loose match by VIN, etc. If left null, the application will attempt to match the fuel transaction to a device and driver at time of an Add or a Set. If no match is found, the fuel transaction’s <code className="small-code-sample">Device</code> and <code className="small-code-sample">Driver</code> properties defaults to NoDevice and UnknownDriver.
+			</li>
+		</ul>
+		<h2>DVIRLog</h2>
+		<ul>
+			<li>
+				<a href="/sdk/software/api/reference/#DVIRLog">DVIRLog</a> - Added <code className="small-code-sample">AuthorityName</code>, <code className="small-code-sample">AuthorityAddress</code>, <code className="small-code-sample">Odometer</code>, <code className="small-code-sample">LoadHeight</code>, <code className="small-code-sample">LoadWidth</code> and <code className="small-code-sample">IsInspectedByDriver</code>. These properties support Canadian DVIR inspections. AuthorityName and AuthorityAddress are automatically populated based on what the user’s corresponding fields are at the time. Odometer currently only applies to the entered <code className="small-code-sample">Hubometer</code> value for Trailer DVIRs.
+			</li>
+		</ul>
+		<h2>ConditionType</h2>
+		<ul>
+			<li>
+				<a href="/sdk/software/api/reference/#ConditionType">ConditionType</a> - Added <code className="small-code-sample">IsValueLessThanPercent</code> and <code className="small-code-sample">IsValueMoreThanPercent</code>. These properties are used to create a percentage threshold for speeding violations, rather than an exact speed value under/over the current posted road speed.
+			</li>
+		</ul>
+		<h2>WebServerInvoker (Nuget Only)</h2>
+		<p>This method has been changed to use generics instead of passing type in, and returning an object, that needs to be cast. For example, <code className="small-code-sample">var version = (string)(await invoker.InvokeAsync("GetVersion", typeof(string)));</code> is now <code className="small-code-sample">var version = await invoker.InvokeAsync&lt;string&gt;("GetVersion");</code></p>
+		<InformationalBox>
+			<p>While not an officially supported component, it’s possible <code className="small-code-sample">WebServerInvoker</code> is being used by some integrations. For this reason we thought it worth mentioning this change.</p>
```

Comment:
The character U+2019 "’" could be confused with the ASCII character U+0060 "`", which is more common in source code